packages/tools/devtools/devtools-view/src/ThemeHelper.ts

```diff
@@ -16,5 +21,10 @@ export function getFluentUIThemeToUse(): Theme {
 		defaultTheme = webDarkTheme;
 	}
 
+	// Add a condition to check for high contrast mode
+	if (window.matchMedia?.("(forced-colors: active)").matches) {
```

Comment:
Can we add a comment linking to the documentation that explains this property we're looking for?